packages/core/src/config/vendure-config.ts

```diff
@@ -401,9 +401,8 @@ export interface AuthOptions {
      * @description
      * Determines whether new User accounts require verification of their email address.
      *
-     * If set to "true", when registering via the `registerCustomerAccount` mutation, one should *not* set the
-     * `password` property - doing so will result in an error. Instead, the password is set at a later stage
-     * (once the email with the verification token has been opened) via the `verifyCustomerAccount` mutation.
+     * If set to "true", the customer will be required to verify their email address using a verification token
+     * they receive in their email. See {@link registerCustomerAccount}.
```

Comment:
Does this @link work when linking to a mutation?

Reply:
No, because the TS compiler does not know about the graphql schema.

Reply:
Ahh ok, I though it was a higher level tool that resolved these links on the docs website. Will remove it